README.md

```diff
@@ -13,7 +13,8 @@
 A framework for autonomous economic agent (AEA) development
 
 <p align="center">
-  <img src="/data/aea.png?raw=true" alt="AEA Description" width="70%"/>
+  <img src="https://raw.githubusercontent.com/fetchai/agents-aea/master/data/aea.png>" 
```

Comment:
Please revert, relative links here are preferable. Substitution to absolute links should happen in `setup.py`